classes/simplemap.php

```diff
@@ -2289,27 +2289,27 @@ function show_toolbar( $title = '' ) {
 			<table class="sm-toolbar" cellspacing="0" cellpadding="0" border="0">
 				<tr>
 					<td class="sm-page-title">
-						<h2><?php _e( $title, 'simplemap' ); ?></h2>
+						<h2><?php _e( $title, 'SimpleMap' ); ?></h2>
 					</td>
 					<td class="sm-toolbar-item">
 						<a href="http://simplemap-plugin.com" target="_blank"
-						   title="<?php _e( 'Go to our website', 'simplemap' ); ?>">SimpleMap <?php _e( 'Home Page', 'simplemap' ); ?></a>
+						   title="<?php _e( 'Go to our website', 'SimpleMap' ); ?>">SimpleMap <?php _e( 'Home Page', 'SimpleMap' ); ?></a>
 					</td>
 					<td class="sm-toolbar-item">
 						<a href="<?php echo admin_url( 'admin.php?page=simplemap-help' ); ?>"
-						   title="<?php _e( 'Premium Support', 'simplemap' ); ?>"><?php _e( 'Premium Support', 'simplemap' ); ?></a>
+						   title="<?php _e( 'Premium Support', 'SimpleMap' ); ?>"><?php _e( 'Premium Support', 'SimpleMap' ); ?></a>
 					</td>
 					<td class="sm-toolbar-item">
-						<a target="_blank" title="<?php _e( 'Donate', 'simplemap' ); ?>"href="https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=mrtorbert%40gmail%2ecom&item_name=SimpleMap&item_number=Support%20Open%20Source&no_shipping=0&no_note=1&tax=0&currency_code=USD&lc=US&bn=PP%2dDonationsBF&charset=UTF%2d8">
-							<img src="<?php echo SIMPLEMAP_URL; ?>/inc/images/donate.jpg" alt="<?php _e( 'Donate with Paypal', 'simplemap' ); ?>"/>
+						<a target="_blank" title="<?php _e( 'Donate', 'SimpleMap' ); ?>"href="https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=mrtorbert%40gmail%2ecom&item_name=SimpleMap&item_number=Support%20Open%20Source&no_shipping=0&no_note=1&tax=0&currency_code=USD&lc=US&bn=PP%2dDonationsBF&charset=UTF%2d8">
+							<img src="<?php echo SIMPLEMAP_URL; ?>/inc/images/donate.jpg" alt="<?php _e( 'Donate with Paypal', 'SimpleMap' ); ?>"/>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 147 characters

<!-- Comment by [SideCI](https://sideci.com) -->